hubmap_commons/type_client.py

```diff
@@ -26,70 +28,98 @@
 #    alt-names: [salmon_rnaseq_bulk]
     alt-names: []
     primary: false
+    contains-pii: false
     vitessce-hints: []
 
 CODEX:
     description: CODEX
     alt-names: []
     primary: true
+    contains-pii: false
     vitessce-hints: []
 
 codex_cytokit:
     description: CODEX [Cytokit + SPRM]
     alt-names: []
     primary: false
+    contains-pii: false
     vitessce-hints: ['codex', 'is_image', 'is_tiled']
 
 image_pyramid:
     description: Image Pyramid
     alt-names: []
     primary: false
+    contains_pii: false
+    vis-only: true
     vitessce-hints: ['is_image', 'pyramid']
 
-SNAREseq:
-    description: SNARE-seq
-    alt-names: []
+SNARE-ATACseq2:
+    description: snATACseq (SNARE-seq2)
+    alt-names: ['SNAREseq', 'SNARE2-ATACseq']
     primary: true
+    contains-pii: true
+    vitessce-hints: []
+
+SNARE-RNAseq2:
+    description: snRNAseq (SNARE-seq2)
+    alt-names: ['SNARE2-RNAseq']
+    primary: true
+    contains-pii: true
     vitessce-hints: []
 
 sc_atac_seq_snare_lab:
-#SNAREseq_lab:
-    description: scATAC-seq (SNARE-seq) [Lab Processed]
-#    alt-names: [sc_atac_seq_snare_lab]
+    description: snATAC-seq (SNARE-seq2) [Lab Processed]
     alt-names: []
     primary: false
     vitessce-hints: []
 
+sc_rna_seq_snare_lab:
+    description: snRNA-seq (SNARE-seq2) [Lab Processed]
+    alt-names: []
+    primary: false
+    vitessce-hints: []
+
+salmon_rnaseq_snareseq:
+    description: snRNA-seq (SNARE-seq2) [Salmon]
+    alt-names: []
+    primary: false
+    contains-pii: false
+    vitessce-hints: ['is_sc', 'rna']
+
 sc_atac_seq_snare:
-#SNAREseq_snapATAC:
-    description: scATAC-seq (SNARE-seq) [SnapATAC]
-#    alt-names: [sc_atac_seq_snare]
+    description: snATAC-seq (SNARE-seq2) [SnapATAC]
     alt-names: []
     primary: false
+    contains-pii: false
     vitessce-hints: ['is_sc', 'atac']
 
 scRNA-Seq-10x:
     description: scRNA-seq (10x Genomics)
     alt-names: []
     primary: true
+    contains-pii: true
     vitessce-hints: []
 
 scRNA-Seq-10x_salmon:
     description: scRNA-seq (10x Genomics) [Salmon]
     alt-names: [salmon_rnaseq_10x]
     primary: false
+    contains-pii: false
     vitessce-hints: ['is_sc', 'rna']
 
 PAS:
     description: PAS Stained Microscopy
     alt-names: []
     primary: true
+    contains-pii: false
     vitessce-hints: []
 
 PAS_pyramid:
     description: PAS Stained Microscopy [Image Pyramid]
     alt-names: [["PAS", "Image Pyramid"], ["Image Pyramid", "PAS"]]
     primary: false
+    contains-pii: false
+    vis-only: true
```

Comment:
I'm concerned with having the entire structure duplicated here. If it's really a dummy, could the data be obviously fake?

Reply:
The dummy client is used for testing, but it is used via unittest.mock.patch .  Any test run against the dummy client can also be run against the actual client, so the dummy client must reply in the same way as the actual client to any query.  That limits the dummy client to valid replies, rather than obviously-fake replies.